incubator/haproxy-ingress/templates/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Do we have to use `namespace` resource? Can we use 'helm install --namespace myns mychart' ?